config.go

```diff
@@ -351,7 +359,9 @@ type Config struct {
 	MaxPendingChannels int    `long:"maxpendingchannels" description:"The maximum number of incoming pending channels permitted per peer."`
 	BackupFilePath     string `long:"backupfilepath" description:"The target location of the channel backup file"`
 
-	FeeURL string `long:"feeurl" description:"Optional URL for external fee estimation. If no URL is specified, the method for fee estimation will depend on the chosen backend and network. Must be set for neutrino on mainnet."`
+	FeeURL              string        `long:"feeurl" description:"Optional URL for external fee estimation. If no URL is specified, the method for fee estimation will depend on the chosen backend and network. Must be set for neutrino on mainnet."`
```

Comment:
Since we are here, how about creating a new group `fee` and put all fee related config in this group? For compatibility, we can use `hidden` flag to deprecate this `--feeurl` and set `--fee.feeurl` to its value if `--feeurl` is used. We can even rename `fee.feeurl` to `fee.url` for clarity.

Reply:
Makes sense. Could we take this opportunity to fully remote the already deprecated `neutrino.feeurl` setting so that we're not maintaining multiple levels of deprecated settings?

Reply:
I've pushed a second commit which refactors to group the fee settings under a new `fee` group.